Core/include/Acts/Propagator/Propagator.hpp

```diff
@@ -128,7 +128,7 @@ struct PropagatorOptions : public PropagatorPlainOptions {
     eoptions.maxRungeKuttaStepTrials = maxRungeKuttaStepTrials;
     eoptions.maxStepSize = maxStepSize;
     eoptions.targetTolerance = targetTolerance;
-    eoptions.pathLimit = direction * std::abs(pathLimit);
+    eoptions.pathLimit = pathLimit;
```

Comment:
It's still not obvious for me that this doesn't break for backwards propagation. The treatment of how the path aborter applies this is unchanged. Is the pathlength accumulation in the aborter signed or unsigned?

Reply:
right this should go again. I attempted to remove the sign of the path length until I realized that I kept it intentionally before

I think this still works because the path limit aborter is very sign agnostic. we apply `abs` multiple times which seems a bit of a too wide contract to me but I would keep it for now